### PR TITLE
python3 ready crossref.py wikicfp.py

### PR DESCRIPTION
**Pull request by andrecastro0o**
Here is the python3 ready.

### CODE REVIEW

File: crossref.py [hunk 1]
```diff
@@ -18,23 +19,33 @@ def handleData(response):
...
-            metadata['proceedings-title'] = ''.join(response['message']['items'][i]['title']).encode('utf-8')
-            metadata['doi'] = response['message']['items'][i]['DOI']
-
-            # TODO: fix encoding
-            [''.join(x).encode('utf-8') for x in metadata ]
-            print (metadata)
-
-            # write metadata to file
-            file.writerow(metadata)
+        print('handleData', response['message'].keys())
```
Comment (lilients) [inline crossref.py:22]: Should this be indented inside the for loop? @andrecastro0o

File: wikicfp.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 startId = sys.argv[1]
 stopId = sys.argv[2]
 
-filename = 'wikicfp-events'
+filename ='wikicfp-events'
```
Comment (lilients) [inline wikicfp.py:19]: Not necessary.

**Review by lilients — DISMISSED** [review]
Thanks! I added two comments. :)

File: crossref.py (hunk 1 above)
Comment (andrecastro0o) [reply crossref.py:22]: no. It is correct like that. is so we see the keys of response['message'], which the loop will loop over

File: wikicfp.py (hunk 2 above)
Comment (andrecastro0o) [reply wikicfp.py:19]: :+1

**Pull request merged**